app/controllers/api/v1/loans_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Api::V1
+  class LoansController < ApplicationController
+    def index
+      @loans = Loan.all
+      render json: @loans, include: %i[deal]
```

Comment:
context on include: %i[deal] ?

Reply:
how does it load data from database, is a separate query or n + 1 ?

Reply:
Yeah, this is a separate query, what I am trying to do is include nested resources (ie. the deal) when I return loans from the api. 
On the front end, many of the tables I use reference the deals.street ex. "232 Mulberry St."
Probably, something I can add to the Loans table to avoid these performance issues altogether.

Reply:
if %i deal causes another query that is not what you want to do.  you can fix this by eager_loading the deal in the query.
`@loans = Loan.includes(:deals).all` <- but don't ever use .all for reasons described above